src/psyclone/tests/test_files/dynamo0p3/infrastructure/argument_mod.F90

```diff
@@ -109,6 +109,13 @@ module argument_mod
   integer, public, parameter :: EVALUATOR           = 507
   integer, public, parameter :: GH_EVALUATOR        = 508
 
+  ! Stencil metadata
+  integer, public, parameter :: XORY1D = 1
+  integer, public, parameter :: X1D    = 2
+  integer, public, parameter :: Y1D    = 3
+  integer, public, parameter :: CROSS  = 4
+  integer, public, parameter :: STENCIL(4) = -1
```

Comment:
P.S. Turns out that stencil initialisation `integer, public, parameter :: STENCIL(4) = -1 ` is in the LFRic code. It would still be good to add a correctness check in PSyclone.

Reply:
`PSyclone/src/psyclone/tests/test_files/dynamo0p3/infrastructure/*` is PSyclone's cut-down version of the LFRic infrastructure which we use to test that PSyclone produces code that will compile. I needed to add this code as we hadn't copied it across. I don't think it makes sense to add correctness checks here.